esmvaltool/diag_scripts/mpqb/mpqb_auxmaps.py

```diff
@@ -6,11 +6,11 @@
 import iris
 import numpy as np
 
-from diag1d import mannkendall1d, theilslopes1d
+from esmvaltool.diag_scripts.shared.trend_mpqb_common.diag1d import mannkendall1d, theilslopes1d
```

Comment:
@bascrezee 
I added the functions missing in this branch to the shared directory, so we can refer to them in the trend package as well.